crates/bevy_color/src/color_ops.rs

```diff
@@ -61,3 +61,39 @@ pub trait Alpha: Sized {
         self.alpha() >= 1.0
     }
 }
+
+/// Method for manipulating the hue of a color.
+pub trait Hue: Sized {
+    /// Return a new version of this color with the hue channel set to the given value.
+    fn with_hue(&self, hue: f32) -> Self;
+
+    /// Return the hue of this color (0.0 - 360.0).
```

Comment:
```suggestion
    /// Return the hue of this color [0.0, 360.0].
```

Reply:
At the same time, I think we shouldn't keep the interval here. What if we find a colorspace that for some reason will go by another interval?

Reply:
Certainly, it's not guaranteed that the range of hue values will fit within this range in every color space.
Thank you for your comment!

Reply:
Yeah, I agree that we don't have many colorspaces out there with some odd behavior, but  at the same time, this trait is made to be generic, and I think that should reflect on docs